packages/components/src/List/ItemTitle/ItemTitle.test.js

```diff
@@ -122,7 +122,30 @@ describe('ItemTitle', () => {
 
 		// when
 		const wrapper = mount(titleInstance);
-		wrapper.find(Button).simulate('click', { stopPropagation: () => {} });
+		wrapper.find(Button).simulate('click', { button: 0, stopPropagation: () => {} });
+
+		// then
+		expect(props.titleProps.onClick).toBeCalled();
+		const callArgs = props.titleProps.onClick.mock.calls[0];
+		expect(callArgs[1]).toBe(item);
```

Comment:
Replace the 3 lines above by `expect(props.titleProps.onClick).toBeCalledWith(item)`